README.md

```diff
@@ -42,6 +42,34 @@ In the installation screen, select a minimum of:
 
 All other components are optional.
 
+#### Local DB
+
+Find out if Local DB is already installed
+
+```Batchfile
+powershell -NoProfile sqllocaldb -?
+```
+
+If the command fails or is not existant then Local DB is not installed or needs to be reconfigured. Follow the steps below:
+
+* Download SQL Express [installer](https://www.microsoft.com/en-us/sql-server/sql-server-editions-express)
+* Select the Download Media option
+* Select LocalDB from the option `WHICH PACKAGE WOULD YOU LIKE TO DOWNLOAD`
+* Install `SqlLocalDB.msi` from the download location
+
+#### SQL CMD
+
+Find out if SQLCMD is already installed
+
+```Batchfile
+powershell -NoProfile sqlcmd -?
+```
+
+If the command fails or is not existant then SQLCMD is not installed or needs to be reconfigured. Follow the steps below:
+
+* Download [Command Line Utilities for SQL Server](https://www.microsoft.com/en-us/download/details.aspx?id=53591) for your platform
+* Install `MsSqlCmdLnUtils.msi`
```

Comment:
Again, this may be too much detail. As with Visual Studio, we could just write:

> Install [Command Line Utilities for SQL Server](https://www.microsoft.com/en-us/download/details.aspx?id=53591).